vscode/src/completions/inline-completion-item-provider.ts

```diff
@@ -435,13 +453,14 @@ export class InlineCompletionItemProvider
                     FeatureFlag.CodyAutocompleteUserLatency
                 ),
             }
-
-            const artificialDelay = getArtificialDelay(
-                latencyFeatureFlags,
-                document.uri.toString(),
-                document.languageId,
-                completionIntent
-            )
+            const params = {
+                featureFlags: latencyFeatureFlags,
+                uri: document.uri.toString(),
+                languageId: document.languageId,
+                codyCompletionDisableLowPerfLangDelay: this.disableLowPerfLangDelay,
+                completionIntent: completionIntent,
+            }
+            const artificialDelay = getArtificialDelay({ ...params })
```

Comment:
This way, the created object is type-checked on the declaration. Also, we don't need to create a new object from `params`.

```suggestion
            const artificialDelay = getArtificialDelay({
                featureFlags: latencyFeatureFlags,
                uri: document.uri.toString(),
                languageId: document.languageId,
                codyCompletionDisableLowPerfLangDelay: this.disableLowPerfLangDelay,
                completionIntent,
            })
```